microservices/netinfra_utils/workers/worker_wrapper.py

```diff
@@ -46,15 +46,12 @@ def register(self, task_type, task_definition=None):
             print('Error while registering task ' + str(err))
 
     def poll_and_execute(self, taskType, exec_function, domain=None):
-        poll_wait = 5000
+        poll_wait = 1000
         while True:
             time.sleep(float(self.polling_interval))
-            print(self.timestamp() + ' Polling for task: ' + taskType + ' with wait ' + str(poll_wait))
```

Comment:
do not just remove print. Better to solve the issue https://github.com/FRINXio/FRINX-machine/issues/109 in separate commit